.github/workflows/rpm-build-and-test.yml

```diff
@@ -297,6 +303,12 @@ jobs:
         with:
           name: ${{ env.DISTRO_NAME }} ${{ env.DISTRO_VERSION }} RPM repository
           path: ${{ env.REPO_PATH}}${{ github.run_number }}/artifact/artifacts/${{ env.TARGET }}
+      - name: Update commit status
+        uses: ouzi-dev/commit-status-updater@v2
```

Comment:
Do we want to pin SHA for new additions?

Reply:
Not IMO.  At least not for the sake of ticking a checkbox.  If we independently decide that that is a burden we want to take on for our own reasons, that is one thing.  But I think this blanket requirement to pin everything to a git hash is going to prove to be unworkable and get some push-back.  Or maybe we are an outlier with over a hundred of these dependencies that we'd have to manually track and update, etc.

Reply:
Like we discussed on #13865, we need more investigation on these things, so no problem from me.